backend/Tests/TreeOfAKind.UnitTests/Validators/ValidatorTestsBase.cs

```diff
@@ -0,0 +1,24 @@
+using System.Reflection;
+using AutoFixture;
+using TreeOfAKind.Application.Command.Trees.AddPerson;
+
+namespace TreeOfAKind.UnitTests.Validators
+{
+    public class ValidatorTestsBase<TValidator> where TValidator : new()
+    {
+        protected readonly TValidator Validator;
+        protected readonly Fixture Fixture = new Fixture();
+
+        public ValidatorTestsBase()
+        {
+            Validator = new TValidator();
+        }
+
+        protected void SetPropertyWithReflection<T, TValue>(T instance, string fieldName, TValue value)
+        {
+            var field = typeof(T).GetField($"<{fieldName}>k__BackingField",
+                BindingFlags.Instance | BindingFlags.NonPublic);
```

Comment:
Oh right, that isn't really how integration tests work

Reply:
Those are unit tests

Reply:
I just don't want to create an object each time and set all those fields one by one :<
I'm just testing a validator, and those only checks if the field is not empty and etc

Reply:
Won't the validator not validate if there are some `null` fields that shouldn't be null?

Reply:
The rest is generated